README.md

```diff
@@ -1,154 +1,139 @@
-# Nerves.Runtime
+# Nerves Runtime
 
 [![Build Status](https://travis-ci.org/nerves-project/nerves_runtime.svg)](https://travis-ci.org/nerves-project/nerves_runtime.svg)
 [![Hex version](https://img.shields.io/hexpm/v/nerves_runtime.svg "Hex version")](https://hex.pm/packages/nerves_runtime)
 
-Nerves.Runtime is a core component of Nerves. It contains applications and
-libraries that are expected to be useful on all Nerves devices.  Here are some
-of its features:
+Nerves Runtime is a core component of Nerves.
+It contains applications and libraries that are expected to be useful on all Nerves devices.
+Here are some of its features:
 
-* Generic system and filesystem initialization (suitable for use with
-  [bootloader](https://github.com/nerves-project/bootloader))
+* Generic system and filesystem initialization (suitable for use with [bootloader](https://github.com/nerves-project/bootloader))
 * Introspection of Nerves system firmware and deployment metadata
-* A custom shell for debugging and running commands in a `bash` shell like
-  environment
+* A custom shell for debugging and running commands in a `bash`-like environment
 * Device reboot and shutdown
-* A small Linux kernel `uevent` application for capturing hardware change events
-  and more
+* A small Linux kernel `uevent` application for capturing hardware change events and more
 * More to come...
 
-The following sections describe the features in more detail. For even more
-information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
+The following sections describe the features in more detail.
+For even more information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
 
-## System initialization
+## System Initialization
```

Comment:
I've always used title case for headings on websites in general (but especially for documentation), so I assumed these uses of sentence case weren't intentional. It doesn't matter to me, except that I think we should be consistent. If you can live with title case, I agree that this is the more common convention.